### PR TITLE
Add concept exercise Pac-Man Rules for booleans

### PR DESCRIPTION
**Pull request by kytrinyx**
This squashes the commits from PR #1093 that was not made against main.

### CODE REVIEW

**Comment by petertseng** [comment]
Recording the commit ID before I make changes: it is https://github.com/exercism/haskell/pull/1099/commits/33ae093e34f01d399f0f1036add491aaa62f77d5

**Review by petertseng — APPROVED** [review]
As you can see, I made some changes:

pacman-rules: fix whitespace errors
4198f1b
packman-rules/config: use correct indentation
bef9f66
pacman-rules/instructions: rm extra underscore
79db5c7
pacman-rules/docs: one sentence per line
af3a7e7
pacman-rules/test: rm extra space in quote
e3ff0b7
pacman-rules/test: Consistently use "a" for "ghost"
d4b85e8
pacman-rules/test: consistently use "while" in eatsGhost
5815bc6 

compare the overall changes via https://github.com/exercism/haskell/compare/33ae093e34f01d399f0f1036add491aaa62f77d5..83535afa52c94a3cdcef47a43592940174059d58

If all good, let's merge. (I will not merge it right now because that would be unilateral unreviewed changes). If any further changes are needed and/or any of my changes are inappropriate, let's hear about them.

File: exercises/concept/pacman-rules/test/Tests.hs
Comment (petertseng) [inline exercises/concept/pacman-rules/test/Tests.hs:7]: and inconsistency that I decided wasn't important enough to resolve, but could be: this test uses the phrase "while powered" and "while un-powered", whereas tests for `loses` and `wins` do not use the word "while"; they just use the words "powered" and "un-powered" alone.

Comment (kytrinyx) [reply exercises/concept/pacman-rules/test/Tests.hs:7]: I like the `while`. I went ahead and updated and force-pushed.

**Comment by kytrinyx** [comment]
All your changes make sense, @petertseng! I think we're good to merge if you are happy with the current state.

**Review by petertseng — APPROVED** [review]

**Comment by petertseng** [comment]
My thanks to everyone who had a hand in this one.

File: exercises/concept/pacman-rules/test/Tests.hs
Comment (petertseng) [reply exercises/concept/pacman-rules/test/Tests.hs:7]: All right! While reading that message I realised there's something implicit in there that I overlooked: as you can see, I'm 100% comfortable with having branches force-pushed even while I potentially am working on them and have them checked out. And I was used to doing that when everything should be squashed, since then nobody has to remember to recheck the message when merging. But I realised not everyone might be the same way. For example I think it's sort of a breach of etiquette for me to have done it without having asked first. So I know it's late, but sorry for doing that without notice.

**Comment by ErikSchierboom** [comment]
Lovely!

**Pull request merged**
